fastlane/testflight.md

```diff
@@ -114,14 +120,14 @@ This step validates most of your six Secrets and provides error messages if it d
 
 ## Create App Group
 
-If you previously built Trio using Mac with Xcode with this Apple ID, skip ahead to [Optional: App Group Description Modification](#optional-app-group-description-modification).
```

Comment:
This is the portion of testflight.md that needs to be updated. It refers to the new Trio App Group.
The earlier diffs should not be included. And only some of the later parts.

Also looks like there may have been some lint applied to some format. Not sure if we are doing that.

Reply:
I will fix this now.
Linting happened on my end, as I edit markdowns in VSCode and it has my work-specific markdown configured.

Reply:
Linting was done via my VSCode. I don't think this should do any issues for us, it linted the file.

I'm not sure what parts of this section needs to be reworked. The section now needs to refer to the new. It looks like you commented on the old code, not the new diff? Not sure here. I've update the section once again and added the italic disclaimer text, hinting at a Trio specific app group. Let me know if this is alright for you.